Accessor not available in dataset

I fetched the latest updates after the lazy_loading PR and ran pytest and am seeing a lot of errors with accessors not loading.  Clean, QCFilter, and QCTests are no longer available in the datasets for some reason.

FAILED test_io.py::test_io_mfdataset - AttributeError: 'Dataset' object has no attribute 'clean'
FAILED test_io.py::test_io_write - AttributeError: 'Dataset' object has no attribute 'clean'
FAILED test_io.py::test_clean_cf_qc - AttributeError: 'Dataset' object has no attribute 'clean'

--- a/act/__init__.py
+++ b/act/__init__.py
@@ -5,11 +5,13 @@ ACT: The Atmospheric Community Toolkit
 """
 
 import lazy_loader as lazy
+
 # No more pandas warnings
 from pandas.plotting import register_matplotlib_converters
 
 from . import tests
 from ._version import get_versions
+from .qc import QCFilter, QCTests, clean
 
 register_matplotlib_converters()
 

--- a/act/qc/__init__.py
+++ b/act/qc/__init__.py
@@ -8,6 +8,8 @@ import lazy_loader as lazy
 
 # We need to import clean first to register the accessor
 from .clean import *
+from .qcfilter import QCFilter
+from .qctests import QCTests
 
 __getattr__, __dir__, __all__ = lazy.attach(
     __name__,

